live-examples/html-examples/text-content/css/figure.css

```diff
@@ -0,0 +1,15 @@
+figure {
+    border: thin #C0C0C0 solid;
+    display: flex;
+    flex-flow: column;
+    margin: 0;
+    padding: 5px;
+}
+
+figcaption {
+    background: #222;
```

Comment:
Nit: Code style: Because you are only specifying a colour, rather be explicit and use the `background-color` property.